tests/test_uniswap.py

```diff
@@ -19,7 +19,7 @@ class GanacheInstance:
     eth_privkey: str
 
 
-@pytest.fixture(scope="module", params=[1, 2])
+@pytest.fixture(scope="module", params=[3])
```

Comment:
Disabled testing v1 and v2 for now. Should be re-enabled before merging (at least v2).